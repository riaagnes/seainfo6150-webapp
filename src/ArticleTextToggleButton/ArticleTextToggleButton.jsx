import React from "react";
import PropTypes from "prop-types";

const ArticleTextToggleButton = ({ updateText, buttonText }) => {
  return (
    <section>
      <p>
        <button onClick={updateText}>{buttonText}</button>
      </p>
    </section>
  );
};

ArticleTextToggleButton.propTypes = {
  article: PropTypes.object.isRequired,
};
export default ArticleTextToggleButton;
