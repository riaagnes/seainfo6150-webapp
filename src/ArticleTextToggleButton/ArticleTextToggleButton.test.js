import React from "react";
import ArticleTextToggleButton from "./ArticleTextToggleButton";

describe("ArticleList tests", () => {
    it("renders correctly", () => {
      const { container } = render(<ArticleTextToggleButton />);
      expect(container).toMatchSnapshot();
    });
});