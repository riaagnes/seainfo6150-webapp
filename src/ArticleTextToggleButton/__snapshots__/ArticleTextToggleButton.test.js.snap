// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleList tests renders correctly 1`] = `
<div>
  <button
    class="button"
  />
</div>
`;
